File: src/WellTool.Cron/hutool-core/src/test/java/cn/hutool/core/util/IssueICA9S5Test.java
package cn.hutool.core.util;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

import java.util.List;

public class IssueICA9S5Test {
	@Test
	public void test() {
		String a = "ENUM{\\ndisable ~ 0\\nenable ~ 1\\n}";
		final List<String> split = StrUtil.split(a, "\\n");
		Assertions.assertEquals(4, split.size());
	}
}

File: src/WellTool.Core.Tests/IssueICA9S5Test.cs
using Xunit;
using WellTool.Core;
using WellTool.Core.Util;
using WellTool.Core.Text;

namespace WellTool.Core.Tests
{
    /// <summary>
    /// Issue #ICA9S5 测试
    /// </summary>
    public class IssueICA9S5Test
    {
        [Fact]
        public void Test()
        {
            string a = "ENUM{\ndisable ~ 0\nenable ~ 1\n}";
            var split = StrSplitter.SplitByString(a, "\n");
            Assert.Equal(4, split.Length);
        }
    }
}
